src/garage/tf/algos/reps.py

```diff
@@ -15,7 +15,8 @@
 from garage.tf.optimizers import LbfgsOptimizer
 
 
-class REPS(BatchPolopt):
+# pylint: disable=differing-param-doc, differing-type-doc
```

Comment:
why?

Reply:
This is because pylint cannot recognize the reference format. I tried to use this format, 
```
References:
  ...
```
But the complaints did not go away.